README.adoc

```diff
@@ -59,6 +59,8 @@ Smallrye JWT supports many properties which can be used to customize the token p
 |smallrye.jwt.expiration.grace|60|Expiration grace in seconds. By default an expired token will still be accepted if the current time is no more than 1 min after the token expiry time.
 |smallrye.jwt.verify.aud|none|Comma separated list of the audiences that a token `aud` claim may contain.
 |smallrye.jwt.required.claims|none|Comma separated list of the claims that a token must contain.
+|smallrye.jwt.encrypt.key-location|none|Config property allows the location of a key to be specified which will be used to encrypt the claims or inner JWT.
```

Comment:
@bobbyphilip Thanks for this PR. I believe we should instead have `==Configuration Properties` sub section added at https://github.com/smallrye/smallrye-jwt/blob/master/doc/modules/ROOT/pages/generate-jwt.adoc as the properties in this section are about dealing with the already secured token (verification/etc). Can you please move them ?

Reply:
Also, we are in the process of moving our documentation to Antora. Right now, this is still duplicate but we hope to have Antora has the main source. Can you also add the documentation here: https://github.com/smallrye/smallrye-jwt/blob/master/doc/modules/ROOT/pages/configuration.adoc?

Thanks!

Reply:
@radcortez I've asked @bobbyphilip to keep these 2 properties scoped in the `generate-jwt.adoc` guide only, the existing config properties about the verification only

Reply:
Ops sorry. @bobbyphilip please do as @sberyozkin requested.

Reply:
@radcortez np, thanks for trying to help here